common-requests/install.md

```diff
@@ -1,19 +1,37 @@
 ---
 layout: page
 title: "Installation"
-description: "TBD"
+description: "Instructions for installing the Open Horizon project's Management Hub, Agent, and CLI all in one VM"
 ---
 
 # Install
 
 ## Pre-requisites
 
-To run the [all-in-one instructions](https://github.com/open-horizon/devops/tree/master/mgmt-hub) for the Management Hub, Agent, and CLI all in one VM, you need to have a VM with at least 4GB RAM and 20GB storage running Ubuntu Server 18.04 LTS.  Below is an example video showing how to install Ubuntu Server on VirtualBox 6.1 on OSX Catalina.
+To run the [all-in-one instructions](https://github.com/open-horizon/devops/tree/master/mgmt-hub) for the Management Hub, Agent, and CLI all in one VM, you need a VM with: 
+
+* >= 4GB RAM 
+* 20GB storage 
+* Ubuntu Server 18.04 LTS
+
+This video shows an example of installing Ubuntu Server on VirtualBox 6.1 on OSX Catalina.
```

Comment:
Sorry if I missed it before, but lead-in sentences usually have closing colons (:).